web/styles/app_components.css

```diff
@@ -466,9 +466,17 @@ div.overlay {
         margin-left: 11px;
         margin-bottom: 5px;
     }
+
+    .input-append .pm-list-filter {
+        /* Input width = 100% - 11px margin x2 - 6px padding x2 - 1px border x2. */
+        width: calc(100% - 36px);
+        margin-left: 11px;
+        margin-bottom: 5px;
+    }
```

Comment:
Reading this code -- is it really the case that `stream-list-filter` has 1px different margin than these others?

We can leave this for a follow-up, but it seems likely we should aim to rewrite these filter elements as a new `sidebar-filter-input` component.

Reply:
Though it should have been but looks like it is not applicable at all on `stream-list-filter` and rather another property of same kind from `left-sidebar.css` is applicable.

For `pm-list-filter` this was actually a bug and there was no need to consider margin at all, I've fixed that!

Reply:
The reason behind the 11px could be that there is an extra `margin-left` of `3px` along with a `padding-left` of `20px` which it is trying to adjust as 11px on both sides (hence x2). I am not sure though why was there that extra margin.